src/components/Modals/Send/views/Address.tsx

```diff
@@ -20,20 +20,25 @@ import { useChainAdapters } from 'context/ChainAdaptersProvider/ChainAdaptersPro
 import { useModal } from 'context/ModalProvider/ModalProvider'
 
 import { AddressInput } from '../AddressInput/AddressInput'
-import { SendFormFields } from '../Form'
+import { SendFormFields, SendInput } from '../Form'
 import { SendRoutes } from '../Send'
 
 export const Address = () => {
   const history = useHistory()
   const translate = useTranslate()
   const {
     formState: { errors }
-  } = useFormContext()
-  const [address, asset] = useWatch({ name: [SendFormFields.Address, SendFormFields.Asset] })
+  } = useFormContext<SendInput>()
+  const address = useWatch<SendInput, SendFormFields.Address>({ name: SendFormFields.Address })
+  const asset = useWatch<SendInput, SendFormFields.Asset>({ name: SendFormFields.Asset })
+
   const chainAdapters = useChainAdapters()
-  const adapter = chainAdapters.byChain(asset.chain)
   const { send } = useModal()
 
+  if (!(asset?.chain && asset?.name && address)) return null
```

Comment:
I don't know if returning `null` is the right thing here vs. some form of placeholder, but I'm no UI dev so I'd prefer to leave that up to someone else.